axelor-human-resource/src/main/java/com/axelor/apps/hr/service/expense/ExpenseValidateService.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Axelor Business Solutions
+ *
+ * Copyright (C) 2005-2023 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+package com.axelor.apps.hr.service.expense;
+
+import com.axelor.apps.base.AxelorException;
+import com.axelor.apps.hr.db.Expense;
+import com.axelor.message.db.Message;
+import java.io.IOException;
+import wslite.json.JSONException;
+
+public interface ExpenseValidateService {
+  public void validate(Expense expense) throws AxelorException;
+
+  public Expense compute(Expense expense);
```

Comment:
I don't get why `compute` is in this service, I think it should be moved to a separate service.